lib/p5.play.js

```diff
@@ -1356,8 +1356,8 @@ function Sprite(pInst, _x, _y, _w, _h) {
     //if has animation get the animation bounding box
     //working only for preloaded images
     if(animations[currentAnimation]) {
-      this._internalWidth = animations[currentAnimation].getWidth()*abs(this._getScaleX());
-      this._internalHeight = animations[currentAnimation].getHeight()*abs(this._getScaleY());
+      this._internalWidth = animations[currentAnimation].getWidth();
+      this._internalHeight = animations[currentAnimation].getHeight();
       this.setCollider('rectangle');
     } else {
       this.setCollider('rectangle');
```

Comment:
Not sure (I haven't been in this code in a while) but I _think_ we could remove these lines altogether because `setCollider('rectangle')` shouldn't depend on the sprite's `_internalWidth` etc. being set anymore - which also removes the if/else here.  Discussed in person, worth investigating.